server/config.js

```diff
@@ -0,0 +1,4 @@
+/** Configuration object for a production virtool UI server*/
+module.exports = {
+    minAPIVersion: "23.4.0",
+};
```

Comment:
* I think this should go in `package.json` under a `virtool` section.
* Change to `minimumApiVersion`

Reply:
Done, but it requires copying `package.json` into the production build image. I don't think that's a big deal, but flagging it just in case